vignettes/howto-dm-rows.Rmd

```diff
@@ -138,7 +141,7 @@ dm_insert_out <-
   dm_rows_insert(dm_insert_in)
 ``````
 
-This gives us a warning that changes will not be persisted.
+This gives us a warning that changes will not persist (i.e., they are temporary).
```

Comment:
Is this still true?

Reply:
Looks like it:

``` r
library(DBI)
library(dm)
library(tidyverse)

parent <- tibble(value = c("A", "B", "C"), pk = 1:3)
child <- tibble(value = c("a", "b", "c"), pk = 1:3, fk = c(1, 1, NA))
demo_dm <-
  dm(parent = parent, child = child) %>%
  dm_add_pk(parent, pk) %>%
  dm_add_pk(child, pk) %>%
  dm_add_fk(child, fk, parent)

sqlite_db <- dbConnect(RSQLite::SQLite())
demo_sql <- copy_dm_to(sqlite_db, demo_dm, temporary = FALSE)

new_parent <- tibble(value = "D", pk = 4)
new_child <- tibble(value = "d", pk = 4, fk = 4)

dm_insert_in <-
  dm(parent = new_parent, child = new_child) %>%
  copy_dm_to(sqlite_db, ., temporary = TRUE)

dm_insert_out <-
  demo_sql %>%
  dm_rows_insert(dm_insert_in)
#> Not persisting, use `in_place = FALSE` to turn off this message.

dbDisconnect(sqlite_db)
```

<sup>Created on 2022-06-19 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>